tests/services/orders/order_status_service/conftest.py

```diff
@@ -22,12 +27,63 @@ def order(store: Store, helpers: StoreHelpers) -> Order:
 
 
 @pytest.fixture
-def analysis_summary(order: Order) -> AnalysisSummary:
-    return AnalysisSummary(
-        order_id=order.id,
-        analysis_id=1,
-        delivered=2,
-        running=3,
-        cancelled=4,
-        failed=5,
+def sample_not_received(store: Store, helpers: StoreHelpers) -> Sample:
+    return helpers.add_sample(
+        store=store,
+        internal_id="not_received",
     )
+
+
+@pytest.fixture
+def sample_in_preparation(store: Store, helpers: StoreHelpers) -> Sample:
+    return helpers.add_sample(
+        store=store,
+        internal_id="in_prep",
+        received_at=datetime.now(),
+    )
+
+
+@pytest.fixture
+def sample_in_sequencing(store: Store, helpers: StoreHelpers) -> Sample:
+    return helpers.add_sample(
+        store=store,
+        internal_id="in_sequencing",
+        received_at=datetime.now(),
+        prepared_at=datetime.now(),
+    )
+
+
+@pytest.fixture
+def order_with_cases(
+    store: Store,
+    helpers: StoreHelpers,
+    order: Order,
+    sample_in_preparation: Sample,
+    sample_in_sequencing: Sample,
+    sample_not_received: Sample,
+) -> Order:
+    case_not_received: Case = helpers.ensure_case(
+        store=store,
+        customer=order.customer,
+        order=order,
+        case_name="case_not_received",
+        case_id="case_not_received",
+    )
+    case_in_preparation: Case = helpers.ensure_case(
+        store=store,
+        customer=order.customer,
+        order=order,
+        case_name="case_in_preparation",
+        case_id="case_in_preparation",
+    )
+    case_in_sequencing: Case = helpers.ensure_case(
+        store=store,
+        customer=order.customer,
+        order=order,
+        case_name="case_in_sequencing",
+        case_id="case_in_sequencing",
+    )
+    helpers.add_relationship(store=store, sample=sample_not_received, case=case_not_received)
+    helpers.add_relationship(store=store, sample=sample_in_preparation, case=case_in_preparation)
+    helpers.add_relationship(store=store, sample=sample_in_sequencing, case=case_in_sequencing)
```

Comment:
Is not the big thing we want to test the interplay that happens when cases have multiple samples with different statuses? Right now I am uncertain if we would catch any such issues.

Reply:
Added additional samples to cover these issues https://github.com/Clinical-Genomics/cg/pull/3050/commits/f015e1623c7a117ed082363c3c1d941173a6e1a0